site/styles/_footer.scss

```diff
@@ -40,7 +42,7 @@ footer {
 		}
 	}
 	.copyright-statement {
-		font-size: 8px;
+		font-size: 12px;
```

Comment:
👍  Always find myself increasing this size because I'm old and can't read tiny print